Mathlib/Combinatorics/Catalan.lean

```diff
@@ -142,10 +142,12 @@ theorem succ_mul_catalan_eq_centralBinom (n : ℕ) : (n + 1) * catalan n = n.cen
   (Nat.eq_mul_of_div_eq_right n.succ_dvd_centralBinom (catalan_eq_centralBinom_div n).symm).symm
 #align succ_mul_catalan_eq_central_binom succ_mul_catalan_eq_centralBinom
 
-theorem catalan_two : catalan 2 = 2 := by unfold catalan; rfl
+theorem catalan_two : catalan 2 = 2 := by
+norm_num [catalan_eq_centralBinom_div, Nat.centralBinom, Nat.choose]
 #align catalan_two catalan_two
 
-theorem catalan_three : catalan 3 = 5 := by unfold catalan; rfl
```

Comment:
Why not keep the `rfl` proof?

Reply:
I assume `norm_num` will eventually become better at computing binomials than the kernel, if it isn't already.
As with most of these proofs I simply restore the mathlib3 proof unless the new one looks to have an advantage